cooperatives/nycode.md

```diff
@@ -18,3 +18,11 @@ parent: Cooperatives
 
 ## Description
 “The C.O.D.E is a data sharing resource for local governments so that they can better inform their blight prevention, mitigation, and resolution strategies.” Schenectady, Amsterdam, and Gloversville partnered, with funding from the New York Department of State. It is a centrally hosted website.
+
+In 2015, the Mayors and Code Enforcement Officials from New York State's cities of Schenectady, Amsterdam and Troy came together to sharedGrid their lessons learned in fighting property blight in their communities. With leadership provided by City of Schenectady's Mayor Gary McCarthy, these conversations turned into a much longer, and larger, discussion of a statewide issue about the inability to sharedGrid code enforcement related information (i.e. problem actors and code enforcement processes) between municipalities on a regular and systematic basis for the purposes of helping inform efforts to prevent and mitigate bight. With Mayor McCarthy leading the way, the group of cities decided to do something about the challenges they and other communities face.
```

Comment:
Is "sharedGrid" a typo? @abbeyns149

Reply:
I was confused by that word too... I checked on their website and this is taken directly from it - I am a bit unsure on what the word means, but it was used in this context, so I don't know if it would be better to leave it or change it?

Reply:
Hm. I'll add [sic] after it to indicate we think this might be a mistake from the source